diff --git a/Makefile.am b/Makefile.am
index 885f1ec..964657a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,36 +1,10 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS=anonymizer \
-debug_sender \
-device_classifier \
-email_reporter \
+SUBDIRS=flow_meter \
 flowcounter \
-json_dump \
-json_replay \
-logger \
-logreplay \
-merger \
-report2idea \
-traffic_repeater \
-endiverter
+logreplay
 
-EXTRA_DIST = AUTHORS COPYING ChangeLog INSTALL NEWS README.md nfreader
-
-if HAVE_LIBNF
-SUBDIRS += nfreader
-else
-if HAVE_LIBNFDUMP
-SUBDIRS += nfreader
-endif
-endif
-
-if HAVE_BISON
-SUBDIRS += unirecfilter
-endif
-
-if HAVE_LIBPCAP
-SUBDIRS += flow_meter
-endif
+EXTRA_DIST = AUTHORS COPYING ChangeLog INSTALL NEWS README.md
 
 RPMDIR = RPMBUILD
 
diff --git a/configure.ac b/configure.ac
index 4491a37..e569622 100644
--- a/configure.ac
+++ b/configure.ac
@@ -38,11 +38,6 @@ AC_DEFINE_DIR(SYSCONFDIR, [sysconfdir], [Path to configuration files])
 AC_DEFINE_DIR(BINDIR, [bindir], [Path to executable files])
 AC_SUBST(BINDIR)
 
-deviceclassifierdatadir=${pkgdatadir}/device_classifier
-AC_DEFINE_DIR(DEVICECLASSIFIERDATADIR, [deviceclassifierdatadir], [Path to SVM data files])
-AC_SUBST(DEVICECLASSIFIERDATADIR)
-AC_SUBST(deviceclassifierdatadir)
-
 RPM_REQUIRES=
 RPM_BUILDREQ=
 
@@ -75,22 +70,10 @@ esac], [repobuild=true])
 
 AX_C_BIGENDIAN_CROSS
 
-AX_OPENMP([], [AC_MSG_ERROR([OpenMP was not found. Some modules need it (e.g. logger and merger)])])
-AC_SUBST(OPENMP_CFLAGS)
-
 AX_LIBTRAP_CHECK
 AX_UNIREC_CHECK
 AX_NEMEACOMMON_CHECK
 
-AC_CHECK_HEADER(idna.h,
-         AC_CHECK_LIB(idn, stringprep_check_version, [libidn=yes], AC_MSG_WARN([libidn not found. The ipdetect module from blacklistfilter will not be compiled.])), AC_MSG_WARN([libidn.h not found. Blacklistfilter modules will not be compiled.]))
-
-AM_CONDITIONAL(HAVE_LIBIDN, test x${libidn} = xyes)
-if [[ -z "$HAVE_LIBIDN_TRUE" ]]; then
-RPM_REQUIRES+=" libidn"
-RPM_BUILDREQ+=" libidn-devel"
-fi
-
 AC_ARG_WITH([flowcachesize],
 	AC_HELP_STRING([--with-flowcachesize=NUMBER],[Set size of flow cache for flow_meter module in number of flow records. Size must be divisible by 32. Default is 65536.]),
 	[
@@ -98,44 +81,6 @@ AC_ARG_WITH([flowcachesize],
 	]
 )
 
-AC_ARG_WITH([nfreader],
-	AC_HELP_STRING([--without-nfreader], [Skip nfreader module.]),
-        [if test "$withval" = "no"; then
-                withnfreader="no"
-        else
-                withnfreader="yes"
-        fi], [withnfreader="yes"])
-
-if test x${withnfreader} = xyes; then
-	# Checks for libraries.
-	AC_CHECK_LIB(nf, lnf_open, [libnf=yes],
-	  [AC_CHECK_LIB(nfdump, nfdump_init, [libnfdump=yes],
-	    [AC_MSG_WARN([libnf and libnfdump not found. The nfreader module will not be compiled.])]
-	  )]
-	)
-
-fi
-AM_CONDITIONAL(HAVE_LIBNFDUMP, test x${libnfdump} = xyes && test x${withnfreader} = xyes)
-if [[ -z "$HAVE_LIBNFDUMP_TRUE" ]]; then
-RPM_REQUIRES+=" libnfdump"
-RPM_BUILDREQ+=" libnfdump-devel"
-fi
-
-AM_CONDITIONAL(HAVE_LIBNF, test x${libnf} = xyes && test x${withnfreader} = xyes)
-if [[ -z "$HAVE_LIBNF_TRUE" ]]; then
-RPM_REQUIRES+=" libnf"
-RPM_BUILDREQ+=" libnf-devel"
-fi
-
-AC_CHECK_PROG(BISON, bison, yes, [""])
-AC_CHECK_PROG(FLEX, flex, yes, [""])
-
-AM_CONDITIONAL(HAVE_BISON, test xyes = x${BISON} -a xyes = x${FLEX})
-if [[ -z "$HAVE_BISON_TRUE" ]]; then
-RPM_BUILDREQ+=" bison"
-RPM_BUILDREQ+=" flex"
-fi
-
 AC_CHECK_HEADER(pcap.h,
         AC_CHECK_LIB(pcap, pcap_open_live, [libpcap=yes], AC_MSG_WARN([libpcap not found. The flow_meter module will not be compiled.])), AC_MSG_WARN([pcap.h not found. The flow_meter module will not be compiled.]))
 
@@ -146,7 +91,7 @@ RPM_BUILDREQ+=" libpcap-devel"
 fi
 
 # Checks for header files.
-AC_CHECK_HEADERS([arpa/inet.h locale.h netdb.h netinet/in.h stddef.h stdint.h stdlib.h string.h sys/socket.h sys/time.h syslog.h unistd.h omp.h])
+AC_CHECK_HEADERS([arpa/inet.h locale.h netdb.h netinet/in.h stddef.h stdint.h stdlib.h string.h sys/socket.h sys/time.h syslog.h unistd.h])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_HEADER_STDBOOL
@@ -187,30 +132,10 @@ AM_CONDITIONAL(MAKE_RPMS, test x$RPMBUILD != x)
 
 # list of all *.in (and Makefile.am) files to process by configure script
 AC_CONFIG_FILES([Makefile
-                 anonymizer/Makefile
-                 debug_sender/Makefile
-                 device_classifier/Makefile
-                 device_classifier/libsvm/tools/train.sh
-                 device_classifier/libsvm/tools/binary.py
-                 device_classifier/libsvm/tools/grid.py
-                 device_classifier/libsvm/Makefile
-                 device_classifier/libsvm/data/Makefile
-                 device_classifier/libsvm/tools/Makefile
-                 email_reporter/Makefile
                  flowcounter/Makefile
                  flow_meter/Makefile
                  flow_meter/tests/Makefile
-                 json_dump/Makefile
-                 json_replay/Makefile
-                 logger/Makefile
                  logreplay/Makefile
-                 merger/Makefile
-                 nemea-modules.spec
-                 nfreader/Makefile
-                 report2idea/Makefile
-                 traffic_repeater/Makefile
-                 unirecfilter/Makefile
-                 endiverter/Makefile
                ])
 
 AC_SUBST(RPM_REQUIRES)
