diff --git a/Makefile.am b/Makefile.am
index 2f71543..2ac92df 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,61 +1,13 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS=aggregator \
-anonymizer \
-debug_sender \
-device_classifier \
-email_reporter \
+SUBDIRS=flow_meter \
+traffic_repeater \
 flowcounter \
-ipv6stats \
-json_dump \
-json_replay \
-link_traffic \
 logger \
 logreplay \
-merger \
-mux \
-demux \
-natpair \
-proto_traffic \
-pdns_exporter \
-report2idea \
-topn \
-traffic_repeater \
-unirec2json \
-endiverter
-
-EXTRA_DIST = AUTHORS COPYING ChangeLog INSTALL NEWS README.md nfreader \
-	debian/README.Debian \
-	debian/changelog \
-	debian/compat \
-	debian/control \
-	debian/copyright \
-	debian/nemea-modules-dev.install \
-	debian/nemea-modules.install \
-	debian/patches \
-	debian/rules \
-	debian/source \
-	debian/watch
-
-if HAVE_LIBNF
-SUBDIRS += nfreader
-SUBDIRS += nfwriter
-endif
+topn
 
-if HAVE_BISON
-SUBDIRS += unirecfilter
-if HAVE_OPENSSL
-SUBDIRS += scalar-aggregator
-endif
-endif
-
-if HAVE_LIBPCAP
-SUBDIRS += flow_meter
-endif
-
-if HAVE_LIBCURL
-SUBDIRS += blooming_history
-endif
+EXTRA_DIST = AUTHORS COPYING ChangeLog INSTALL NEWS README.md
 
 RPMDIR = RPMBUILD
 
@@ -77,16 +29,3 @@ endif
 
 rpm-clean:
 	rm -rf $(RPMDIR)
-
-if MAKE_DEB
-.PHONY: deb
-deb:
-	make distdir && cd nemea-modules-@VERSION@ && debuild -i -us -uc -b
-else
-endif
-
-deb-clean:
-	rm -rf nemea-modules_*.build* nemea-modules_*.changes nemea-modules*.deb nemea-modules_*.orig.tar.gz nemea-modules-*.tar.gz nemea-modules-@VERSION@
-
-clean-local: rpm-clean deb-clean
-
diff --git a/configure.ac b/configure.ac
index 7e6f7ab..e1c0cfa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.63])
-AC_INIT([nemea-modules], [2.8.2], [nemea@cesnet.cz])
+AC_INIT([nemea-modules], [2.6.1], [nemea@cesnet.cz])
 AC_CONFIG_SRCDIR([flowcounter/flowcounter.c])
 AC_CONFIG_HEADERS([config.h])
 RELEASE=1
@@ -43,11 +43,6 @@ AC_SUBST(BINDIR)
 bashcompldir=${sysconfdir}/bash_completion.d
 AC_SUBST(bashcompldir)
 
-deviceclassifierdatadir=${pkgdatadir}/device_classifier
-AC_DEFINE_DIR(DEVICECLASSIFIERDATADIR, [deviceclassifierdatadir], [Path to SVM data files])
-AC_SUBST(DEVICECLASSIFIERDATADIR)
-AC_SUBST(deviceclassifierdatadir)
-
 RPM_REQUIRES=
 RPM_BUILDREQ=
 
@@ -64,7 +59,6 @@ AC_CHECK_PROG(PYTHON, python, python, [""])
 AC_SUBST(PYTHON)
 # Check for rpmbuild
 AC_CHECK_PROG(RPMBUILD, rpmbuild, rpmbuild, [""])
-AC_CHECK_PROG(DEBUILD, debuild, debuild, [""])
 AX_PTHREAD([LIBS="$PTHREAD_LIBS $LIBS"
     CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
     CXXFLAGS="$CXXFLAGS $PTHREAD_CFLAGS"
@@ -81,15 +75,13 @@ esac], [repobuild=false])
 
 AX_C_BIGENDIAN_CROSS
 
-AX_OPENMP([], [AC_MSG_ERROR([OpenMP was not found. Some modules need it (e.g. merger)])])
-AC_SUBST(OPENMP_CFLAGS)
-
 backup_libs=${LIBS}
 AX_LIBTRAP_CHECK
 AX_UNIREC_CHECK
 AX_NEMEACOMMON_CHECK
 LIBS=${backup_libs}
 
+
 AC_ARG_WITH([openssl],
         [AS_HELP_STRING([--without-openssl], [Force to disable openssl])],
         [if test x$withval = xyes; then
@@ -104,22 +96,7 @@ if test x$have_openssl = xyes; then
   RPM_BUILDREQ+=" openssl-devel"
 else
   AC_DEFINE([HAVE_OPENSSL], [0], [Define to 1 if the openssl is available])
-fi
-
-AC_ARG_WITH([libcurl],
-        [AS_HELP_STRING([--without-libcurl], [Force to disable libcurl])],
-        [if test x$withval = xyes; then
-        PKG_CHECK_MODULES([libcurl], [libcurl], [have_libcurl="yes"], [have_libcurl="no"])
-        fi],
-        [PKG_CHECK_MODULES([libcurl], [libcurl], [have_libcurl="yes"], [have_libcurl="no"])])
-
-AM_CONDITIONAL([HAVE_LIBCURL], [test x$have_libcurl = xyes])
-if test x$have_libcurl = xyes; then
-  AC_DEFINE([HAVE_LIBCURL], [1], [Define to 1 if the libcurl is available])
-  RPM_REQUIRES+=" libcurl"
-  RPM_BUILDREQ+=" libcurl-devel"
-else
-  AC_DEFINE([HAVE_LIBCURL], [0], [Define to 1 if the libcurl is available])
+  AC_MSG_WARN([openssl not found. The aggregator module will not be compiled.])
 fi
 
 AC_ARG_WITH([flowcachesize],
@@ -135,36 +112,6 @@ AC_ARG_WITH([flowcachestats],
 	]
 )
 
-AC_ARG_WITH([nfreader],
-	AC_HELP_STRING([--without-nfreader], [Skip nfreader module.]),
-        [if test "$withval" = "no"; then
-                withnfreader="no"
-        else
-                withnfreader="yes"
-        fi], [withnfreader="yes"])
-
-if test x${withnfreader} = xyes; then
-	# Checks for libraries.
-	AC_CHECK_HEADER(libnf.h, AC_CHECK_LIB(nf, lnf_open, [libnf=yes],
-	  [AC_MSG_WARN([libnf not found. The nfreader module will not be compiled.])]
-	), AC_MSG_WARN([libnf.h not found. The nfreader module will not be compiled.]))
-fi
-
-AM_CONDITIONAL(HAVE_LIBNF, test x${libnf} = xyes && test x${withnfreader} = xyes)
-if [[ -z "$HAVE_LIBNF_TRUE" ]]; then
-RPM_REQUIRES+=" libnf"
-RPM_BUILDREQ+=" libnf-devel"
-fi
-
-AC_CHECK_PROG(BISON, bison, yes, [""])
-AC_CHECK_PROG(FLEX, flex, yes, [""])
-
-AM_CONDITIONAL(HAVE_BISON, test xyes = x${BISON} -a xyes = x${FLEX})
-if [[ -z "$HAVE_BISON_TRUE" ]]; then
-RPM_BUILDREQ+=" bison"
-RPM_BUILDREQ+=" flex"
-fi
-
 AC_CHECK_HEADER(pcap.h,
         AC_CHECK_LIB(pcap, pcap_open_live, [libpcap=yes], AC_MSG_WARN([libpcap not found. The flow_meter module will not be compiled.])), AC_MSG_WARN([pcap.h not found. The flow_meter module will not be compiled.]))
 
@@ -175,7 +122,7 @@ RPM_BUILDREQ+=" libpcap-devel"
 fi
 
 # Checks for header files.
-AC_CHECK_HEADERS([arpa/inet.h locale.h netdb.h netinet/in.h stddef.h stdint.h stdlib.h string.h sys/socket.h sys/time.h syslog.h unistd.h omp.h])
+AC_CHECK_HEADERS([arpa/inet.h locale.h netdb.h netinet/in.h stddef.h stdint.h stdlib.h string.h sys/socket.h sys/time.h syslog.h unistd.h])
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_HEADER_STDBOOL
@@ -212,67 +159,18 @@ RPM_RELEASE=1
 AC_SUBST(RPM_RELEASE)
 AM_CONDITIONAL(MAKE_RPMS, test x$RPMBUILD != x)
 
-AM_CONDITIONAL(MAKE_DEB, test x$DEBUILD != x)
-
 #DX_INIT_DOXYGEN([nemea-cpd], [Doxyfile], [doc])
 
 # list of all *.in (and Makefile.am) files to process by configure script
 AC_CONFIG_FILES([Makefile
-                 aggregator/Makefile
-                 anonymizer/Makefile
-                 blooming_history/Makefile
-                 debug_sender/Makefile
-                 device_classifier/Makefile
-                 device_classifier/libsvm/tools/train.sh
-                 device_classifier/libsvm/tools/binary.py
-                 device_classifier/libsvm/tools/grid.py
-                 device_classifier/libsvm/Makefile
-                 device_classifier/libsvm/data/Makefile
-                 device_classifier/libsvm/tools/Makefile
-                 email_reporter/Makefile
                  flowcounter/Makefile
                  flow_meter/Makefile
-                 flow_meter/flow_meter.bash
                  flow_meter/tests/Makefile
-                 ipv6stats/Makefile
-                 json_dump/Makefile
-                 json_replay/Makefile
-                 link_traffic/Makefile
-                 link_traffic/munin/Makefile
-                 link_traffic/link_traff2json.py
+                 flow_meter/flow_meter.bash
                  logger/Makefile
                  logreplay/Makefile
-                 merger/Makefile
-                 mux/Makefile
-                 demux/Makefile
-                 natpair/Makefile
-                 nemea-modules.spec
-                 nfreader/Makefile
-                 pdns_exporter/Makefile
-                 proto_traffic/Makefile
-                 proto_traffic/munin/Makefile
-                 report2idea/Makefile
-                 report2idea/amplificationdetector/Makefile
-                 report2idea/bruteforce/Makefile
-                 report2idea/ddos/Makefile
-                 report2idea/dnstunnel/Makefile
-                 report2idea/haddrscan/Makefile
-                 report2idea/hoststats/Makefile
-                 report2idea/blacklist/Makefile
-                 report2idea/minerdetector/Makefile
-                 report2idea/sipbruteforce/Makefile
-                 report2idea/venom/Makefile
-                 report2idea/voipfraud/Makefile
-                 report2idea/vportscan/Makefile
-                 scalar-aggregator/Makefile
                  topn/Makefile
                  traffic_repeater/Makefile
-                 unirec2json/Makefile
-                 unirecfilter/lib/Makefile
-                 unirecfilter/lib/liburfilter.pc
-                 unirecfilter/Makefile
-                 endiverter/Makefile
-                 nfwriter/Makefile
                ])
 
 AC_SUBST(RPM_REQUIRES)
